setup.py

```diff
@@ -6,7 +6,7 @@
 
 from setuptools import Extension, setup
 
-NO_EXTENSIONS = bool(os.environ.get("MULTIDICT_NO_EXTENSIONS"))
+NO_EXTENSIONS = os.environ.get("MULTIDICT_NO_EXTENSIONS", "0") == "1"
```

Comment:
Same as the previous comment. Although, it doesn't matter much now, since I'll align the packaging with frozenlist and yarl soon.